src/rpm/headers/header.rs

```diff
@@ -441,8 +445,77 @@ impl Header<IndexTag> {
     }
 
     #[inline]
-    pub fn get_install_time(&self) -> Result<i64, RPMError> {
-        self.get_entry_i64_data(IndexTag::RPMTAG_INSTALLTIME)
+    pub fn get_vendor(&self) -> Result<&str, RPMError> {
+        self.get_entry_string_data(IndexTag::RPMTAG_VENDOR)
+    }
+
+    #[inline]
+    pub fn get_url(&self) -> Result<&str, RPMError> {
+        self.get_entry_string_data(IndexTag::RPMTAG_URL)
+    }
+
+    #[inline]
+    pub fn get_license(&self) -> Result<&str, RPMError> {
+        self.get_entry_string_data(IndexTag::RPMTAG_LICENSE)
+    }
+
+    // TODO: internationalized strings
+    // #[inline]
+    // pub fn get_summary(&self) -> Result<&str, RPMError> {
+    //     self.get_entry_string_array_data(IndexTag::RPMTAG_SUMMARY)
+    // }
+
+    // #[inline]
+    // pub fn get_description(&self) -> Result<&str, RPMError> {
+    //     self.get_entry_string_array_data(IndexTag::RPMTAG_DESCRIPTION)
+    // }
+
+    // #[inline]
+    // pub fn get_group(&self) -> Result<&str, RPMError> {
+    //     self.get_entry_string_array_data(IndexTag::RPMTAG_GROUP)
+    // }
+
+    #[inline]
+    pub fn get_packager(&self) -> Result<&str, RPMError> {
+        self.get_entry_string_data(IndexTag::RPMTAG_PACKAGER)
+    }
+
+    #[inline]
+    pub fn get_build_time(&self) -> Result<i64, RPMError> {
+        // Typically this is stored as an i32, however, exposing it as such is a bad idea.
```

Comment:
Annoyingly, `rpm` seems to store this as `i32` but then they claim that it's year 2038 safe because they treat it like a u32 everywhere.  I'm sure that's true, but it's a mess.  Let's not do that.

Reply:
Can we just make it an int64 without causing any further incompatibilites?

Reply:
Do you mean the value of the IndexData itself?

Reply:
`rpm` may in fact treat *all* integers as unsigned despite the name https://github.com/rpm-software-management/rpm/blob/master/docs/manual/tags.md#scalar-types

Reply:
My largest concern is if existing implementations assume fixed types per tag, such that other implementations would read only individual bytes of i.e. a larger integer.

Reply:
I've asked for some clarification on how the types are supposed to work, because there's a lot of inconsistency.  The docs claim the integers are unsigned but the signatures and types suggest otherwise.

Reply:
I've filed https://github.com/rpm-rs/rpm/issues/41

> <_ffesti_> and yes all integer tags are unsigned

Reply:
I assume IRC? We should document these impl details..